iree/hal/vulkan/vulkan_device.cc

```diff
@@ -227,6 +224,7 @@ StatusOr<ref_ptr<VulkanDevice>> VulkanDevice::Create(
     VkPhysicalDevice physical_device,
     const ExtensibilitySpec& extensibility_spec,
     const ref_ptr<DynamicSymbols>& syms,
+    bool force_timeline_semaphore_emulation,
```

Comment:
we'll want to have an options struct for this kind of stuff - probably worth doing now so we don't grow more later

Reply:
I was going to wait until a second option like this was added, but ok :P